src/gql/generated/types.ts

```diff
@@ -480,25 +480,6 @@ export type LogMessage = {
   version?: Maybe<Scalars["Int"]>;
 };
 
-export type LogkeeperBuild = {
```

Comment:
Can you make sure to run `yarn codegen` while checking out the most recent `main` branch in Evergreen? I don't think this should be getting deleted.

Reply:
done